src/login/70-uaccess.rules.in

```diff
@@ -80,4 +80,7 @@ ENV{ID_SOFTWARE_RADIO}=="?*", TAG+="uaccess"
 # 3D printers, CNC machines, laser cutters, 3D scanners, etc.
 ENV{ID_MAKER_TOOL}=="?*", TAG+="uaccess"
 
+# Protocol analyzers
+ENV{ID_SIGNAL_ANALYZER}=="?*", TAG+="uaccess"
```

Comment:
i still would feel more comfortable if `ENV{DEVTYPE}=="usb_device"` was added here too. Why? because with the hwdb matches this PR adds the ID_SIGNAL_ANALYZER=1 property will end up on both the `usd_device` and the `usb_interface` kobjects. Which I think is fine. But I think the "uaccess" should only end up on the former, not the latter. Simply to optimize what logind/uaccess logic has to listen for.